tests/test_collection.py

```diff
@@ -169,51 +166,47 @@ def test_index_info(self):
         db.test.save({})  # create collection
         ix_info = yield db.test.index_information()
         self.assertEqual(len(ix_info), 1)
-
-        self.assert_("_id_" in ix_info)
+        self.assertEqual(ix_info["_id_"]["name"], "_id_")
 
         yield db.test.create_index(filter.sort(filter.ASCENDING("hello")))
         ix_info = yield db.test.index_information()
         self.assertEqual(len(ix_info), 2)
-        
-        self.assertEqual(ix_info["hello_1"], [("hello", 1)])
+        self.assertIsNotNone(ix_info["hello_1"]["name"] == "hello_1")
```

Comment:
`assertIsNotNone(a == b)` will always pass since both True and False are not None :)